src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -322,13 +322,13 @@
   "Download": "Download",
   "DownloadClient": "Download Client",
   "DownloadClientCheckNoneAvailableHealthCheckMessage": "No download client is available",
-  "DownloadClientCheckUnableToCommunicateWithHealthCheckMessage": "Unable to communicate with {0}.",
+  "DownloadClientCheckUnableToCommunicateWithHealthCheckMessage": "Unable to communicate with {0}. Check your logs for details.",
   "DownloadClientOptionsLoadError": "Unable to load download client options",
-  "DownloadClientRootFolderHealthCheckMessage": "Download client {0} places downloads in the root folder {1}. You should not download to a root folder.",
+  "DownloadClientRootFolderHealthCheckMessage": "Download client {0} downloads completed downloads in to the root folder {1}. You should not download to the app's Library ('root') folder.",
```

Comment:
Too many downloads, `places` is more correct.

`You should not download to the {2}'s root folder.` - where `{2}` is Sonarr/Radarr/Lidarr/etc